itou/static/js/city_autocomplete_field.js

```diff
@@ -2,13 +2,14 @@ $(document).ready(() => {
 
   let citySearchInput = $('.js-city-autocomplete-input')
   let hiddenCityInput = $('.js-city-autocomplete-hidden')
-
+  let searchButton = $('.btn')
```

Comment:
Ici il faut être plus spécifique dans le sélecteur sinon tu risques d'appliquer le code JS à tous les éléments de la page avec une classe `.btn` qui est très courante dans Bootstrap.

En général, directement dans le HTML, j'ajoute une classe spécifique aux éléments qui commence par `.js-` comme tu peux le voir sur les lignes juste au dessus. Ça permet 2 choses :

- d'être plus spécifique
- de savoir immédiatement en lisant le HTML qu'un bout de code JS est attaché à cet élément

Reply:
J'avais hésité à faire ça. Merci du retour

Reply:
Bof non pas de raison spéciale si ce n'est de pouvoir réutiliser https://github.com/CSSLint/csslint/wiki/disallow-ids-in-selectors

Reply:
Ok, ben je ferais comme ça alors